docs/source/connecting/connecting.md

```diff
@@ -34,6 +34,52 @@ async fn main() -> Result<(), Box<dyn Error>> {
 After successfully connecting to some specified node the driver will fetch topology information about
 other nodes in this cluster and connect to them as well.
 
+Scylla Serverless is an elastic and dynamic deployment model. When creating a `Session` you need to
+specify the secure connection bundle as follows:
+
+```rust
+# extern crate scylla;
+# extern crate tokio;
+# fn check_only_compiles() {
+use std::path::Path;
+use std::error::Error;
+use scylla::CloudSessionBuilder;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let session = CloudSessionBuilder::new(Path::new("config_data.yaml"))
+        .unwrap()
+        .build()
+        .await
+        .unwrap();
+
+    Ok(())
+}
+# }
+```
+
+Here is an example of a configuration file for a serverless cluster:
```

Comment:
I think it would be good to mention here that the configuration file is provided by us, not something that the user is supposed to write themself.

Are there any docs, tutorials or other learning resources that explain the concept of the "connection bundle"? If so, then we could link them here to guide users that might not be familiar with serverless.

Reply:
I added a note about the configuration file but unfortunately, I could not find any specific tutorial or documentation explaining the concept of a bundle file and that it will be provided after creating a serverless cluster.